tests/acceptance/run.sh

```diff
@@ -517,9 +514,12 @@ fi
 # If the caller did not mention specific tags, skip the skipped tests by default
 if [ "${BEHAT_TAGS_OPTION_FOUND}" = false ]
 then
+	if [[ -z $BEHAT_FILTER_TAGS ]]
+	then
+		BEHAT_FILTER_TAGS="~@skip"
 	# If the caller has already specified specifically to run "@skip" scenarios
 	# then do not append "not @skip"
-	if [[ ! ${BEHAT_FILTER_TAGS} =~ "&&@skip&&" ]]
+	elif [[ ! ${BEHAT_FILTER_TAGS} =~ "(^|&)@skip(&|$)" ]]
```

Comment:
I think it should be without quote
```suggestion
	elif [[ ! ${BEHAT_FILTER_TAGS} =~ (^|&)@skip(&|$) ]]
```